db/schema.rb

```diff
@@ -88,6 +100,13 @@
   add_index "customers_labels", ["customer_id"], name: "index_customers_labels_on_customer_id"
   add_index "customers_labels", ["label_id"], name: "index_customers_labels_on_label_id"
 
+  create_table "identifications", force: :cascade do |t|
+    t.string   "provider"
+    t.string   "uid"
```

Comment:
What is the point of identification table with authorization table existing?

Reply:
Its probably an old table from a very old branch that somehow stayed in. I'll write a migration to delete it.

Reply:
Then delete it